rasa-ext-vim/src/Rasa/Ext/Vim.hs

```diff
@@ -83,10 +83,10 @@ anyMode _ = return ()
 
 -- | Listeners for keypresses when in 'Insert' mode
 insert :: [Keypress] -> BufAction ()
-insert [KEsc] = mode .= Normal
+insert [KEsc _] = mode .= Normal
 
-insert [KBS] = moveRangesByN (-1) >> delete
-insert [KEnter] = insertText "\n" >> moveRangesByC (Coord 1 0) >> startOfLine
+insert [KBS _] = moveRangesByN (-1) >> delete
+insert [KEnter _] = insertText "\n" >> moveRangesByC (Coord 1 0) >> startOfLine
```

Comment:
Guess it is just a personal preference :). I don't know how close you want to mirror vim with Rasa.Ext.Vim. But vim accepts enter with modifiers.